### PR TITLE
Fixes compile changelog action. Hopefully.

### PR DESCRIPTION
**Pull request by AnturK**
Let's see how it goes.

### CODE REVIEW

**Comment by SpaceManiac** [comment]
I warned you dog

**Comment by msgerbs** [comment]
Can somebody ELI5 why those spaces caused this?

**Comment by SpaceManiac** [comment]
> Can somebody ELI5 why those spaces caused this?

The `[` command in bash is not special syntax, it's literally a command named `[`. You have to use spaces to separate commands from their arguments.

**Comment by msgerbs** [comment]
> 
> 
> > Can somebody ELI5 why those spaces caused this?
> 
> The `[` command in bash is not special syntax, it's literally a command named `[`. You have to use spaces to separate commands from their arguments.

Ahh. Deceptively simple mistake with big consequences.
